### PR TITLE
[PR-15477] Integration documentation for Loyalty

### PR DESCRIPTION
**Pull request by dgtv**
## Demo

https://deploy-preview-241--talkable-docs.netlify.app/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-15477)

### CODE REVIEW

File: source/integration/loyalty/integration_components.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+   **Script Location.** The Initialization script should be placed in the head template or some template
+   that spans every page. All other integration components are dependent on the Init script.
+
+   **Data Capture** The Initialization Script should pass variables for logged in users:
+
+    - Email
+
+2. :ref:`Loyalty Dashboard <integration/loyalty/integration_components/loyalty_dashboard>`
+
+3. :ref:`Loyalty Redeem Widget <integration/loyalty/integration_components/loyalty_redeem_widget>`
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:23]:
```suggestion
2. :ref:`Loyalty Dashboard`

3. :ref:`Loyalty Redeem Widget`
```

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-sections-in-the-same-document

File: source/integration/loyalty/integration_components.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+.. code-block:: html
+
+   <div id="talkable-loyalty"></div>
+
+Loyalty Dashboard Notes
+-----------------------
+
+1. The :ref:`Talkable Initialization Script <integration/loyalty/integration_components/initialization_script>` must be present in your head
+   template in order for the loyalty dashboard to work
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:83]:
```suggestion
1. The `Talkable Initialization Script <Initialization Script_>`_ must be
   present in your head template in order for the Loyalty Dashboard to work.
```

File: source/integration/loyalty/integration_components.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+---------------------------
+
+1. **Site ID.** You can obtain your Site ID by logging into the Talkable
+   platform where Site ID is displayed on your Dashboard and URL as seen
+   here:
+
+   .. figure:: /_static/img/site_id.png
+      :alt: Site ID
+
+2. **Variables.** Use your dynamic variables to pass user details
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:62]:
```suggestion
2. **Variables.** Use your dynamic variables to pass user details.
```

File: source/integration/loyalty.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+   .. figure:: /_static/img/loyalty/loyalty_checkout.png
+      :alt: Sample checkout
+
+4. Define Loyalty points redemption batches.
+
+   Each redemption batch determines progressive point redemption options based on the available points balance of the loyalty member. For example: 500 points would equal a discount coupon for $40 off, 1000 points would equal $100 and so on.
+
+   For every redemption batch please generate single-use coupons for a certain discount amount ($40 off, $100 off, etc) and upload all coupons lists into Talkable loyalty coupon lists.
+
+   It is recommended to upload 100K single-use coupons into every coupon list.
```
Comment (zhuravel) [inline source/integration/loyalty.rst:56]: Non-technical people might not know that K represents thousand.
```suggestion
   It is recommended to upload 100,000 single-use coupons into every coupon list.
```

File: source/integration/loyalty/integration_components.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+|hr|
+
+.. _integration/loyalty/integration_components/loyalty_dashboard:
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:66]:
```suggestion
.. _Loyalty Dashboard:
```

File: source/integration/loyalty/integration_components.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+   dashboard can be added inline inside your user accounts menu, however
+   the dashboard content width for proper display is 980px.
+
+3. Allow your users to reach the Loyalty Dashboard by adding a link
+   from any place that makes sense considering your website configuration.
+   Most common uses are links in the user accounts section, or from the user accounts menu.
+
+|hr|
+
+.. _integration/loyalty/integration_components/loyalty_redeem_widget:
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:96]:
```suggestion
.. _Loyalty Redeem Widget:
```

File: source/integration/loyalty/integration_components.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+.. meta::
+   :description: Find the components of what Talkable is composed.
+
+Integration Components
+======================
+
+Talkable Loyalty is composed of the following components:
+
+1. :ref:`Initialization Script <integration/loyalty/integration_components/initialization_script>`
```
Comment (zhuravel) [inline source/integration/loyalty/integration_components.rst:12]:
```suggestion
1. `Initialization Script <Initialization Script_>`_
```

**Review by zhuravel — CHANGES_REQUESTED** [review]

File: source/integration/loyalty/integration_components.rst [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+   **Script Location.** The Initialization script should be placed in the head template or some template
+   that spans every page. All other integration components are dependent on the Init script.
+
+   **Data Capture** The Initialization Script should pass variables for logged in users:
```
[the remark this answers is not in the document]
Comment (zhuravel) [reply source/integration/loyalty/integration_components.rst]: @dgtv Please fix original as well ↑

**Review by zhuravel — APPROVED** [review]

File: source/integration/loyalty.rst [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+   :description: You can integrate Talkable Loyalty to e-commerce sites in a few quick steps.
+
+Loyalty Integration (beta)
+==========================
+
+Integrating Talkable Loyalty to e-commerce sites is done in
+a few quick steps by adding the below scripts to your site.
+
+Integration
+-----------
```
Comment (zhuravel) [inline source/integration/loyalty.rst:14]:
```suggestion
.. raw:: html

   <h2>Integration</h2>
```

File: source/integration/loyalty.rst [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+   .. figure:: /_static/img/loyalty/redemption_batches.png
+      :alt: Redemption batches
+
+ .. note::
+
+    All design and loyalty campaign setup is done inside the Talkable platform,
+    then via iframe loyalty campaign content is injected into the site.
+
+Requirements
+------------
```
Comment (zhuravel) [inline source/integration/loyalty.rst:69]:
```suggestion
.. raw:: html

   <h2>Requirements</h2>
```

**Review by zhuravel — COMMENTED** [review]
The only way to make _Integration Components_ not nested under _Requirements_ is to not add _Integration_ and _Requirements_ to tree:

File: source/integration/loyalty.rst (hunk 10 above)
Comment (dgtv) [reply source/integration/loyalty.rst:69]: 👍 Cool, thanks

**Review by zhuravel — APPROVED** [review]

**Pull request merged**